src/test/java/ServerTest.java

```diff
@@ -0,0 +1,11 @@
+// import org.junit.Test;
```

Comment:
Why are the tests commented out? How do you know that your Server class is working correctly?